src/table.rs

```diff
@@ -876,4 +949,22 @@ mod tests {
 
         println!("{table}");
     }
+    #[test]
+    fn test_rows_sorting() {
+        let mut table = Table::new();
+        table
+            .set_header(vec![Cell::new("Names")])
+            .add_row(vec![Cell::new("Alexander the Great")])
+            .add_row(vec![Cell::new("Shahd Aldahar")])
+            .add_row(vec![Cell::new("Neil Armstrong")])
+            .add_row(vec![Cell::new("Hassan Ibn Al-haitham")])
+            .add_row(vec![Cell::new("Al-khwarizmi")]);
+        table.sort_rows(|a, b| b.cells()[0].cmp(&a.cells()[0]));
+        println!("\nsort in descending order:\n");
+        println!("{table}");
+
+        table.sort_rows(|a, b| a.cells()[0].cmp(&b.cells()[0]));
+        println!("\nsort in ascending order:\n");
+        println!("{table}");
+    }
 }
```

Comment:
You added a unit test that doesn't do anything.

Please write proper integration tests that actually test stuff.

Reply:
I was confused about how I could test it. I printed it to be sure is sorted without problems.